fixtures/flight/server/region.js

```diff
@@ -202,13 +206,22 @@ if (process.env.NODE_ENV === 'development') {
 
       const sourceMap = nodeModule.findSourceMap(requestedFilePath);
       let map;
-      // There are two ways to return a source map depending on what we observe in error.stack.
-      // A real app will have a similar choice to make for which strategy to pick.
-      if (!sourceMap || Error.prepareStackTrace === undefined) {
-        // When --enable-source-maps is enabled, the error.stack that we use to track
-        // stacks will have had the source map already applied so it's pointing to the
-        // original source. We return a blank source map that just maps everything to
-        // the original source in this case.
+      if (requestedFilePath.startsWith('node:')) {
```

Comment:
We could filter out node internals on the server for thrown errors [like we do for other owner stacks](https://github.com/facebook/react/blob/main/packages/react-server/src/ReactFlightServer.js#L130).

However, the rationale for filtering those out is mainly to save on bandwidth and `eval()` cost for stacks that are eagerly generated even if there is no error. If it wasn't for that it's generally up to the client to filter these out.

For errors that have actually thrown, that's less of a concern and it's nice to have the additional information if needed.

To make these automatically filtered out we can generate a source map for the internals with the `ignoreList` specified so that it's hidden by default but can still be expanded if you need it for some reason.

Unfortunately ignore lists aren't applied to the default printing of reportError/console.error(error) but that's a general problem that also applies to client infra stacks.